frontend/src/features/admin-form/settings/SettingsEmailsPage.tsx

```diff
@@ -0,0 +1,44 @@
+import { FormControl, Skeleton } from '@chakra-ui/react'
+
+import { FormResponseMode } from '~shared/types'
+
+import FormLabel from '~components/FormControl/FormLabel'
+import { TagInput } from '~components/TagInput'
+
+import { CategoryHeader } from './components/CategoryHeader'
+import { EmailFormSection } from './components/EmailFormSection'
+import { useAdminFormSettings } from './queries'
+
+const AdminEmailSection = (): JSX.Element => {
+  const { data: settings } = useAdminFormSettings()
+
+  const isEmailOrStorageMode =
+    settings?.responseMode === FormResponseMode.Email ||
+    settings?.responseMode === FormResponseMode.Encrypt
+
+  if (settings && isEmailOrStorageMode) {
+    return <EmailFormSection settings={settings} />
+  } else {
+    return <EmailFormSectionSkeleton />
```

Comment:
This looks like we'll render a skeleton if it is not `Email` or `Encrypt` mode. I understand that the `<SettingsEmailPage />` is not rendered if it is neither Email nor Encrypt, but it may not be true if there was a mistake in rendering.

Would prefer that we render `null` if it is not `isEmailOrStorageMode`, and render `<EmailFormSectionSkeleton />` if `settings` is not yet ready.

Reply:
Resolved! Changed it to return `false` if  `!isEmailOrStorageMode` since it seems that it's a better approach than `return null`

> Instead of using return null, a better approach is to use return false. When a component returns false, React treats it as if it had returned null, but it also tells React not to render any of the component's children. 
[From here](https://medium.com/@davidkelley87/stop-using-return-null-in-react-a2ebf08fc9cd)